### PR TITLE
added subtitle

### PR DESCRIPTION
**Pull request by ryanlogsdon**

### CODE REVIEW

**Review by tanayagarflyr — APPROVED** [review]
Looks Good!

File: titles.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ Harry Potter and the Soccerer's Stone
 The Stand
 
 Animal Farm
-Ikigai
```
Comment (jameschen56) [inline titles.txt:5]: Why did you remove this line?

File: titles.txt [hunk 2]
```diff
@@ -2,4 +2,5 @@ Harry Potter and the Soccerer's Stone
...
\ No newline at end of file
+Ikigai: The Japanese Secret to a Long and Happy Life
```
Comment (tanayagarflyr) [inline titles.txt:5]: title size is too large

File: titles.txt [hunk 3]
```diff
@@ -2,4 +2,5 @@ Harry Potter and the Soccerer's Stone
 The Stand
```
Comment (jameschen56) [inline titles.txt:2]: Please remove this line!

File: titles.txt (hunk 1 above)
Comment (ryanlogsdon) [reply titles.txt:5]: to add in the subtitle
